utils/extractor/src/index.ts

```diff
@@ -80,7 +82,19 @@ export const Extractor = (
             await asyncBatch(
               capture[sheet.name].data,
               async (chunk) => {
-                await api.records.insert(sheet.id, chunk)
+                const body = zlib.gzipSync(JSON.stringify(chunk))
+                await axios.post(
+                  `${process.env.FLATFILE_API_URL}/v1/sheets/${sheet.id}/records`,
```

Comment:
Correct me if I'm wrong, if `FLATFILE_API_URL` isn't defined in `.env`, Fern defaults to the production url? So if a client will need to add `FLATFILE_API_URL` to their `.env` in order to avoid `Extraction failed connect ECONNREFUSED ::1:80` errors using Axios.

Reply:
That's probably the case! I can update this to default to the production URL if the env variable is not provided.